test/lib.bash

```diff
@@ -200,15 +200,15 @@ function run_rolling_upgrade_tests {
   serving_image_version=$(versions.major_minor "${KNATIVE_SERVING_VERSION}")
   eventing_image_version="${KNATIVE_EVENTING_VERSION}"
   eventing_kafka_image_version=$(versions.major_minor "${KNATIVE_EVENTING_KAFKA_VERSION}")
-  eventing_kafka_broker_image_version=$(versions.major_minor "${KNATIVE_EVENTING_KAFKA_BROKER_VERSION}")
+  eventing_kafka_broker_image_version="${KNATIVE_EVENTING_KAFKA_BROKER_VERSION}"
 
   # mapping based on https://github.com/openshift/release/blob/master/core-services/image-mirroring/knative/mapping_knative_v1_2_quay
   base="quay.io/openshift-knative/"
   image_template=$(
     cat <<-EOF
 $base{{- with .Name }}
 {{- if eq .      "wathola-kafka-sender"}}{{.}}:v$eventing_kafka_image_version
-{{- else if eq . "kafka-consumer"      }}knative-eventing-kafka-broker-test-kafka-consumer:knative-v$eventing_kafka_broker_image_version
+{{- else if eq . "kafka-consumer"      }}knative-eventing-kafka-broker-test-kafka-consumer:$eventing_kafka_broker_image_version
```

Comment:
```
 docker pull quay.io/openshift-knative/knative-eventing-kafka-broker-test-kafka-consumer:f24550ef10c9deea57c2306db05d813406ee73b8
```
didn't work when I used `f24550ef10c9deea57c2306db05d813406ee73b8` as the `eventing_kafka_broker_image_version`

Reply:
Yes, image mirroring for tag by commit is not yet done